lib/offsite_payments/integrations/bit_pay.rb

```diff
@@ -23,12 +23,19 @@ class Helper < OffsitePayments::Helper
         def initialize(order_id, account, options)
           super
           @account = account
-
           add_field('posData', {'orderId' => order_id}.to_json)
           add_field('fullNotifications', true)
           add_field('transactionSpeed', 'high')
         end
 
+        if options[:checkout_token].length >=44
+            self.invoicing_url = 'https://bitpay.com/invoices'
+            #check for characters and revert
+            if options[:checkout_token].include? "0" || options[:checkout_token].include? "O" || options[:checkout_token].include? "I" || options[:checkout_token].include? "l"
+              self.invoicing_url = 'https://bitpay.com/api/invoice'
+            end
```

Comment:
Can you add a test for it? I'd like to understand what's the actual difference between the two type of tokens.